Demo/Demo/ViewController.swift

```diff
@@ -1,13 +1,76 @@
 import UIKit
+import SwiftUI
 import Components
 
-class ViewController: UIViewController {
+class FeatureNavigationController: WKCanvasViewController, WKFeatureNavigationHostingControllerDelegate {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        print(Components().text)
-    }
+	override func viewDidLoad() {
+		super.viewDidLoad()
 
+		let featureNavigationController = WKFeatureNavigationHostingController(delegate: self)
```

Comment:
I may be missing a clear benefit, but for this simple example I would avoid the additional feature-specific view controller layer with `WKFeatureNavigationHostingController` and instantiate it like:

```
let featureNavigationController = WKComponentHostingController(rootView: WKFeatureNavigationView(delegate: self))
addComponent(featureNavigationController, pinToEdges: true, respectSafeArea: true)
```